<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QDialog" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>380</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>540</width>
    <height>380</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>540</width>
    <height>380</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>واژه یاب</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="imageLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>520</width>
       <height>170</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>520</width>
       <height>170</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;IRNazanin&quot;;</string>
       </property>
       <property name="text">
        <string>نام کاربری</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="userLineEdit">
       <property name="inputMask">
        <string comment="*"/>
       </property>
       <property name="placeholderText">
        <string>Username</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;IRNazanin&quot;;</string>
       </property>
       <property name="text">
        <string>گذرواژه</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="passLineEdit">
       <property name="placeholderText">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="loginButton">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;IRNazanin&quot;;</string>
       </property>
       <property name="text">
        <string>ورود</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>userLineEdit</tabstop>
  <tabstop>passLineEdit</tabstop>
  <tabstop>loginButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
